buildSrc/src/main/groovy/com/github/okbuilds/core/model/JavaLibTarget.groovy

```diff
@@ -22,6 +23,17 @@ class JavaLibTarget extends JavaTarget {
         } else {
             retrolambda = null
         }
+
+        // SqlDelight
+        if (project.plugins.hasPlugin('com.squareup.sqldelight')) {
+            File sqldelightDir = new File(project.projectDir, "build/generated/source/sqldelight")
```

Comment:
why is this the same as the one the gradle plugin generated sources end up in?

Can we not use a temp directory for okbuck purposes only?